driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/ODBFeatures.java

```diff
@@ -107,7 +108,7 @@ public boolean supportsComputer() {
 
         @Override
         public boolean supportsTransactions() {
-            return false;
+            return transactionalGraph;
```

Comment:
I would need to learn a little bit more, but, I think features should reflect what we do or do not support.

That affect how the test suites from gremlin gremlin behave.

But, one thing last days thought me, ain't that simple to get gremlin suite running orientdb